apps/pie-storybook/docblocks/component-status.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
"Jamie, why not `tsx`?" I hear you ask.

Good question, originally this was using `tsx`! Unfortunately it seemed to create a React error when serving during the production build both locally and on the preview deploy. The React error was:

`Cannot read properties of null (reading 'useRef')` and seemed to come from some internal React script run inside Storybook (I wasn't calling any React methods). I wondered if it could have been related to our React wrapper components, but I could not figure it out.

I would prefer us to use `tsx` here but I cannot figure out what causes the internal React error. 

If anyone has any ideas, please let me know and we can try to convert back to `tsx` ❤️